client/src/components/ContentNavigator/index.ts

```diff
@@ -247,6 +250,71 @@ class ContentNavigator implements SubscriptionProvider {
           "workbench.actions.treeView.contentdataprovider.collapseAll",
         );
       }),
+      commands.registerCommand(
+        "SAS.convertSasnbToFlw",
+        async (resource: ContentItem) => {
+          const resourceUri = getUri(resource);
+          // Ensure that the command is only available for .sasnb files
+          if (resourceUri.path.endsWith(".sasnb")) {
+            // Open window to chose the name and location of the new .flw file
+            const name = await window.showInputBox({
+              prompt: "Enter the name for the new .flw file",
+              value: resource.name.replace(".sasnb", ".flw"),
+            });
+
+            if (!name) {
+              // User canceled the input box
+              return;
+            }
+
+            if (!name.endsWith(".flw")) {
+              window.showErrorMessage(Messages.InvalidFlwFileNameError);
+              return;
+            }
+
+            // check that name does not exist
+            const parent = await this.contentDataProvider.getParent(resource);
+            const children = await this.contentDataProvider.getChildren(parent);
+            if (children.some((child) => child.name === name)) {
+              window.showErrorMessage(Messages.FileAlreadyExistsError);
+              return;
+            }
```

Comment:
We could skip this check and use `NewFileCreationError` after calling `createFile` (we do this elsewhere in code)

Reply:
Done in 048298e018a51e572e6320a9e44ad08515e02ae5.
It seems that the .flw files are not "seen" anymore by VSCode after the SAS Studio association step.
SAS Studio also handles duplicated file names by adding a suffix.